test/unit/fadd_nbi.c

```diff
@@ -29,6 +29,7 @@
 #include <stdio.h>
 #include <shmem.h>
 #include <shmemx.h>
```

Comment:
We can remove `shmemx.h`? 

Also, is this the only test in the repo that uses `shmemx_wtime()`?

Reply:
> We can remove `shmemx.h`?

Yes, good catch.

> Also, is this the only test in the repo that uses `shmemx_wtime()`?

Oops - I just noticed there are ~4 other tests that do something like this:
```
#ifndef HAVE_SHMEMX_WTIME
static double shmemx_wtime(void) {
    struct timeval tv;
    gettimeofday(&tv, NULL);
    return (double) tv.tv_sec + (double) tv.tv_usec / 1000000.0;
}
#endif /* HAVE_SHMEMX_WTIME */
```

So it seems we can have all these 5 tests use the new header or I can just add this to the `fadd_nbi.c` test.  Any preference @wrrobin?